docs/sdk/serverless-signing/zero-to-hero.md

```diff
@@ -0,0 +1,181 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Zero to Hero
+
+## Prerequisites
+
+- Familiarity with JavaScript
+
+## Overview
+In this section we're gonna cover an end-to-end example from minting a PKP through the ContractsSDK, setting its scope & signing a message with it inside a Lit Action.
+
+Installed the latest contracts-sdk on `cayenne`
+
+```bash
+yarn install @lit-protocol/contracts-sdk@cayenne
+```
+
+## Set up the controller
+
+To initialize a LitContracts client you need an Eth Signer. This can be a standard Ethers wallet or also a `PKPEthersWallet` (more info on the later [here](../authentication/session-sigs/auth-methods/add-remove-auth-methods)). But here, we're gonna use the standard Ethers wallet.
+
+## Initialize the ContractsSDK
+
+We're using the ContractsSDK for the minting the PKP & interacting with it (updating the scopes). So let's initialize it
```

Comment:
consider dividing each of these up into clear steps for the developer ('the first step is to initialize the contracts sdk')